src/implementations/twig/components/description-list/description-list.story.js

```diff
@@ -9,66 +7,59 @@ import dataDescriptionListHorizontal from '@ecl/specs-component-description-list
 import descriptionList from './description-list.html.twig';
 import notes from './README.md';
 
-const prepareList = (data) => {
-  if (data.variant) {
-    data.variant = select(
-      'variant',
-      [data.variant],
-      data.variant,
-      tabLabels.required
-    );
-  }
-  data.items.forEach((item, i) => {
-    if (Array.isArray(item.term)) {
-      item.term.forEach((termItem, j) => {
-        data.items[i].term[j] = text(
-          `items[${i}].term[${j}]`,
-          termItem,
-          tabLabels.required
-        );
-      });
-    } else {
-      item.term = text(`items[${i}].term`, item.term, tabLabels.required);
-    }
-    if (Array.isArray(item.definition)) {
-      item.definition.forEach((definitionItem, k) => {
-        data.items[i].definition[k] = text(
-          `items[${i}].definition[${k}]`,
-          definitionItem,
-          tabLabels.required
-        );
-      });
-    } else {
-      item.definition = text(
-        `items[${i}].definition`,
-        item.definition,
-        tabLabels.required
-      );
-    }
-  });
-
-  getExtraKnobs(data);
+const getArgTypes = (data) => {
+  return {
+    term: {
+      name: 'first item term',
+      type: { name: 'string', required: true },
+      defaultValue: data.items[0].term,
+      description: 'Test the content for the first item term',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '' },
+        category: 'Content',
+      },
+    },
+    description: {
+      name: 'first item definition',
+      type: { name: 'string', required: true },
+      defaultValue: data.items[0].definition,
+      description: 'Test the content for the first item definition',
+      table: {
+        type: { summary: 'string' },
+        defaultValue: { summary: '' },
+        category: 'Content',
+      },
+    },
+  };
+};
 
-  return data;
+const prepareData = (data, args) => {
+  return Object.assign(data, args);
```

Comment:
In this case we cannot use `Object.assign()` like this, what it does is to "merge" the two objects, but the properties of  `args` are not the ones we want to apply to data, here we have to assign them to `data.items[0]`.
So, like:
```
const prepareData = (data, args) => {
  data.items[0].term = args.term;
  data.items[0].definition = args.definition;
  return data;
};
```